### PR TITLE
Added inkscape modules

### PR DESCRIPTION
**Pull request by Zedonboy**
Added inkscape modules so users can easily work with the files from their IDE

### CODE REVIEW

**Comment by Neon22** [comment]
putting all the extensions code in here seems like doing it backwards ??

The idea was you can use this as a template when making new inx and py files for an inkscape extension.
It has not been updated for the upcoming 1.0 release.
See gitlab extensions sub repo for up to date code of the main extensions code.
https://gitlab.com/inkscape/extensions

**Review by oxr463 — CHANGES_REQUESTED** [review]
These should not go in this repository.

It would be better to document installing the extensions locally and then requiring as needed.

File: template.py [hunk 1]
```diff
@@ -4,7 +4,8 @@
 Example of extensions template for inkscape
 
 '''
-
+import sys
+sys.path.insert(0, "./inkscape_pymodules")
```
Comment (Neon22) [inline template.py]: I don't agree with putting msoft visual studio cruft into anything.
please remove

**Comment by Neon22** [comment]
the added codes don't belong here. If they did then every single extension would bloat the system with hundreds of duplicate folders with the same code in it.
So rejecting

**Pull request closed without merging**